python/dask_cudf/dask_cudf/tests/test_struct.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import cudf
+
+import dask_cudf
+
+
+@pytest.mark.parametrize(
+    "data, column",
+    [
+        (
+            {
+                "a": [{"a": [1, 2, 3, 4], "b": "Hello world"}, {}, {"a": []}],
+                "b": [1, 2, 3],
+                "c": ["rapids", "cudf", "hi"],
+            },
+            "a",
+        ),
+        (
+            {"a": [{}, {}, {}], "b": [1, 2, 3], "c": ["rapids", "cudf", "hi"]},
+            "a",
+        ),
+        (
+            {
+                "a": [{}, {}, {}],
+                "b": [{"a": 1}, {"b": 5}, {"c": "Hello"}],
```

Comment:
Would it be difficult to include converage for null elements in the struct column?
I just noticed that the cudf repr may be broken in this case.

Reply:
do you mean something like 
```
{
  "a": [{"a": [1, 2, 3, 4], "b": "Hello world"}, None, {"a": []}],
  "b": [1, 2, 3],
  "c": ["rapids", "cudf", "hi"]
}
```

Reply:
Yep! Exactly what I mean

Reply:
oh hmmm weird, the result wasn't broken, but I think the return fields were scrambled to become `'0', '1'`, rather than the correct `'a', 'b'` and etc. I think this is an issue with remapping the `struct` field names when returning it?